docs/configuration.md

```diff
@@ -746,7 +746,7 @@ rewrite.rules = [AsciiSortImports]
 import foo.{~>, `symbol`, bar, Random}
 ```
 
-### Trailing commas
+## Trailing commas
```

Comment:
i added this section under the rewrite rules, because it actually mutates the ast. i am ok with extracting it to a higher-level section, as long as it's close to the rewrite group.

Reply:
👌 reverted